web-app/src/services/authService.ts

```diff
@@ -20,9 +21,17 @@ export class AuthService {
     const protocol = req.headers["x-forwarded-proto"] || req.protocol;
     const host = req.headers["x-forwarded-host"] || req.get("host");
 
+    // Encrypt
+    const state = {
+      pull_url: req.query.pull_url,
+      github_client_id: this._appConfig.github_client_id,
+    };
```

Comment:
This isn't a random string as suggested [here](https://docs.github.com/en/developers/apps/building-github-apps/identifying-and-authorizing-users-for-github-apps#1-request-a-users-github-identity):

> This should contain a random string to protect against forgery attacks and could contain any other arbitrary data.